app/Http/Controllers/Accessories/AccessoryCheckoutController.php

```diff
@@ -77,6 +77,14 @@ public function store(Request $request, $accessoryId)
             'note' => $request->input('note'),
         ]);
 
+        $checkedout=DB::table('accessories_users')->where('accessory_id', '=', $accessory->id)->count();
+        $available=DB::table('accessories')->where('id', '=', $accessory->id)->first('qty');
```

Comment:
Couldn't we just use `numRemaining()` here?

https://github.com/snipe/snipe-it/blob/c24b398b6f4c768e2531e119235f1013351075c5/app/Models/Accessory.php#L339-L346